app/src/main/kotlin/com/bitwarden/authenticator/data/authenticator/manager/model/VerificationCodeItem.kt

```diff
@@ -13,24 +14,11 @@ package com.bitwarden.authenticator.data.authenticator.manager.model
  */
 data class VerificationCodeItem(
     val code: String,
-    val totpCode: String,
     val periodSeconds: Int,
     val timeLeftSeconds: Int,
     val issueTime: Long,
     val id: String,
-    val username: String?,
```

Comment:
So I did some renaming around `username` because I found that property name to be pretty confusing and not relatable to the TOTP spec. `label` is more appropriate. 

The authenticator app UI does have a way to update the "username" field of an entity, but ultimatly that gets mapped to the TOTP equivalent of `label`.

Reply:
Thanks for the clarification 😸 

All that makes sense, I think. I went ahead and walked back my name change, updating to `accountName` and reinstating the `label` property! 

efc2e54e057938e935f54f4c1607cafff7a2ae30

Reply:
Thanks for that!